.devcontainer/docker-compose.yml

```diff
@@ -83,6 +83,6 @@ services:
   google-storage:
```

Comment:
please add a volume like we do for the other service containers (everything is stored in the `/storage` directory)

Reply:
done.